src/app/shop/items.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { items, IItem, IPrices, ITag } from './data';
+@Injectable({
+  providedIn: 'root',
+})
+export class ItemsService {
+  protected items: IItem[];
+  
+  constructor() {
+    this.items = items;
```

Comment:
Porque sigues marcando el comentario como resuelto sin hacerle nada? this.items es exactamente lo mismo que items por lo que no necesitas en absoluto eso

Reply:
el linter no deja de hacerlo otra manera, exige que este declarado como miembro de la Clase